index.html

```diff
@@ -2942,8 +2942,9 @@ <h3>Zero-Knowledge Proofs</h3>
         <p>
 A zero-knowledge proof is a cryptographic method where an entity can prove to
 another entity that they know a certain value without disclosing the actual
-value. A real-world example is proving that an accredited university has granted
-a degree to you without revealing your identity or any other personally
+value. This is sometimes referred to as selective disclosure in attribute based
+credential systems. A real-world example is proving that an accredited university has
```

Comment:
Hmm, this reordering now reads to me as equating selective disclosure with zero knowledge proofs. A reader could be confused into think they are the same; most critically, selective disclosure involves sharing a particular value (and nothing else) as opposed to a ZKP which involves proving one knows a value without sharing anything else (including not sharing the value itself). Selective disclosure protocols may involve ZKPs, of course, but I think the language here needs some tweaking.

Reply:
Yeah I agree with your reading of it. I wanted to highlight that ZKPs are likely being used for selective disclosure which is what led to this language update. However, your point stands that ZKPs and selective disclosure mechanisms are not the same thing, although they often can be conflated. I'll give an attempt at rewording this in order to make sure this point isn't lost on the reader.

Reply:
I just dropped the additional sentence I had originally added, so that this reads the same as it used to. I figured this would be the easier solution to address your concern. If you have a suggestion to further enhance and untie these concepts in a second round of review I'm happy to add that in as well.